website/source/docs/platform/k8s/connect.html.md

```diff
@@ -319,23 +333,167 @@ connectInject:
+```
+
+### Injection Defaults
+
+If you wish for the injector to always inject, you can set the default to `true`
+in the Helm chart:
+
+```yaml
+connectInject:
+  enabled: true
+  default: true
+```
+
+You can then exclude specific pods via annotation:
+
+```yaml
+annotations:
+  "consul.hashicorp.com/connect-inject": "false"
+```
+
+### Controlling Injection Via Namespace
+
+You can control which Kubernetes namespaces are allowed to be injected via
+the `k8sAllowNamespaces` and `k8sDenyNamespaces` keys:
+
+```yaml
+connectInject:
+  enabled: true
+  k8sAllowNamespaces: ["*"]
+  k8sDenyNamespaces: []
+```
+
+In the default configuration (shown above), services from all namespaces are allowed
+to be injected. Whether or not they're injected depends on the value of `connectInject.default`
+and the `consul.hashicorp.com/connect-inject` annotation.
+
+If you wish to only enable injection in specific namespaces, you can list only those
+namespaces in the `k8sAllowNamespaces` key. In the configuration below
+only the `my-ns-1` and `my-ns-2` namespaces will be enabled for injection.
+All other namespaces will be ignored, even if the connect inject [annotation](#consul-hashicorp-com-connect-inject)
+is set.
+
+```yaml
+connectInject:
+  enabled: true
+  k8sAllowNamespaces: ["my-ns-1", "my-ns-2"]
+  k8sDenyNamespaces: []
+```
+
+If you wish to enable injection in every namespace *except* specific namespaces, you can
+use `*` in the allow list to allow all namespaces and then specify the namespaces to exclude in the deny list:
+
+```yaml
+syncCatalog:
+  enabled: true
+  k8sAllowNamespaces: ["*"]
+  k8sDenyNamespaces: ["no-sync-ns-1", "no-sync-ns-2"]
+```
+
+-> **NOTE:** The deny list takes precedence over the allow list. If a namespace
+is listed in both lists, it will **not** be synced. 
+
+~> **NOTE:** The `kube-system` and `kube-public` namespaces will never be injected.
+
+
+### Consul Clients Required
+
+Connect injection requires that local client agents
+are running on each Kubernetes node. These client agents must be joined to a Consul
+server cluster.
+The Consul server cluster can run either in or out of a Kubernetes cluster.
+
+### Consul Enterprise Namespaces
+
+Consul Enterprise supports Consul namespaces. When Kubernetes pods are registered
+into Consul, you can control which Consul namespace they are registered into.
+
+There are three options available:
+
+1. **Single Destination Namespace** – Register all Kubernetes pods, regardless of namespace,
+into the same Consul namespace.
+
+    This can be configured with:
+    
+    ```yaml
+    global:
+      enableConsulNamespaces: true
+
+    connectInject:
+      enabled: true
+      consulNamespaces:
+        consulDestinationNamespace: "my-consul-ns"
+    ```
+
+       -> **NOTE:** If the destination namespace does not exist we will create it.
+
+1. **Mirror Namespaces** - Register each Kubernetes pod into a Consul namespace with the same name as its Kubernetes namespace.
```

Comment:
```suggestion
2. **Mirror Namespaces** - Register each Kubernetes pod into a Consul namespace with the same name as its Kubernetes namespace.
```

Reply:
Markdown automagically increases the numbers for you. This makes it easy to re-order lists without having to redo all the numbers.